ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -67,6 +68,8 @@ def llmobs_set_tags(
                 llmobs_integration = "bedrock"
             elif model_provider.startswith(OPENAI_PROVIDER_NAME):
                 llmobs_integration = "openai"
+            elif model_provider.startswith(ANTHROPIC_PROVIDER_NAME):
```

Comment:
```suggestion
            elif operation == "chat" and model_provider.startswith(ANTHROPIC_PROVIDER_NAME):
```
Since our anthropic integration doesn't trace LLM calls (only chat model)